### ISSUE TITLE
Does not work with py3 / sage 9.0

### ISSUE THREAD

**Issue by mkoeppe** [sagemath/sage_sample#37]

**Comment by VivianePons** [comment]
Indeed, i am trying to update my own package and I will fix this one as well. Is there any example of a python3 sage package that I could look at?

**Comment by VivianePons** [comment]
I have some issues with Sphinx conf file especially

**Comment by zerline** [comment]
Do you have ```unicode(some_string)``` in your ```conf.py```? If so, you can

```from six import text_type```

and replace your _unicode_ occurrences by _text_type_

**Comment by mkoeppe** [comment]
Unfortunately sage_sample appears to be unmaintained. It is even unclear who is the maintainer.

Here's a package that works: https://github.com/mkoeppe/sage-numerical-interactive-mip

**Comment by egourgoulhon** [comment]
In case this is still usefull (sorry I am discovering this thread and especially Viviane message only today), I have adapted the Sage package kerrgeodesic_gw (https://github.com/BlackHolePerturbationToolkit/kerrgeodesic_gw) to make it work with Python 3 and Sage >= 9.0. This package was constructed from sage_sample, so I guess the changes made to it can be applied to sage_sample too.
